tests/CMakeLists.txt

```diff
@@ -188,6 +188,12 @@ add_test(NAME test_ip_secondary
          COMMAND ${PROJECT_SOURCE_DIR}/tests/ipsecondary_test.py
          --build-dir ${PROJECT_BINARY_DIR} --src-dir ${PROJECT_SOURCE_DIR})
 
+if(BACKEND_FAILURE_TEST)
```

Comment:
Why the new switch? Does the test require new exotic dependencies (that's the usual reason we have these)? If not it can be enabled by default or with another combination of switches.

Reply:
The goal here is to avoid running this test by default since it takes over 700s to run it and enable this test only on CI or if it's instructed by an user. I am open for any alternatives to what is here.

Reply:
I don't think it's really necessary to disable it by default if it's just 700s and it's run by default on CI.

A softer method would just be to add a "long" label to the test so that it can be skipped when running `ctest -LE long`:

    set_tests_properties(test_xxx PROPERTIES LABELS "long")